src/content/guide/how-to-build-a-product/authentication.md

```diff
@@ -0,0 +1,423 @@
+authenticate and setup their devices. This will likely depend on how you
+envision your customers interacting with their connected product.
+  * **Mobile**: Use an iOS or Android mobile app to allow your customers to authenticate, setup their devices, and interact with their product. [More info](/guide/how-to-build-a-product/mobile-app/)
+  * **Web**: Use a web browser and HTTP to allow your customers to authenticate, setup their devices, and interact with their product (*Coming soon*).
+
+## Simple Authentication
+
+As the title suggests, Simple Authentication is the simplest and most straightforward to implement.
+This is because in this method, your application will not have its own
+server/back-end architecture. Instead, your web or mobile app will hit
+the Particle API directly for both session management and device
+interactions. Below is a diagram communicating how simple authentication
+works at a high level:
+
+![Simple authentication with Particle](/assets/images/simple-auth-high-level.png)
+<p class="caption">Your application interacts directly with the Particle
+API using Simple Authentication</p>
+
+Let's take a simple example. Imagine you are the creator of a smart
+lightbulb that can be controlled via a smartphone app. The *customer*,
+or the end-user of the product, uses the mobile app to create an
+account. Behind the scenes, your mobile app hits the Particle API
+directly to create a customer. Then the customer goes through the setup
+process and links their device to their customer account. Again, your
+mobile app uses the Particle API to successfully claim the device to the customer
+account. After the device is setup, the customer can toggle a light on
+and off with the mobile app. This works as your app is able to call
+functions on the customers device using the customer's access token.
+
+All of this is able to happen without the need to have your own server.
+All communiation flows from the mobile client to the Particle cloud,
+then down to the customer's device.
+
+### Advantages of Simple Auth
+
+Simple auth is ideal for getting a Particle product working quickly and
+efficiently. With the lack of your own back-end, development time to
+creating an app to work with a Particle device is shortened. In
+addition, Particle's [mobile SDKs](/reference/ios/) and [JavaScript
+SDK](/reference/javascript/) will handle much of
+the heavy lifting for you when it comes to session management and device
+interaction. In short, simple auth is...simple.
+
+Another advantage of simple authentication is the ability to hide
+Particle from your customers. The SDKs allow for [front-end skinning and
+customization](/reference/ios/#customization) that will allow you to
+create your own brand experience for customers of your app. All
+interaction with Particle will happen behind the scenes, hidden from
+your customers (unless they are tech savvy enough to monitor the network
+traffic to and from your app).
+
+### Disadvantages of Simple Auth
```

Comment:
In the disadvantages section, I'd recommend a change of language/tone. Both the examples *are* actually possible. It's just that we don't help you with them. You have to code them separately your self. Statements like, "this would not be possible" are factually incorrect. Keep in mind that they could still have a web app that is *not involved in authentication* to do all these things. No specific suggestions for language, but suggest going over this section again.